server/sonar-server/src/test/java/org/sonar/server/authentication/UserIdentityAuthenticatorTest.java

```diff
@@ -70,7 +69,6 @@
   @Rule
   public DbTester db = DbTester.create(system2);
 
-  private UserDbTester userTester = new UserDbTester(db);
```

Comment:
I think replacing call to constructor by `db.users()` is definitely a good idea but I don't agree on duplication `db.users()` everywhere in the code. This is noise.